src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchRules.groovy

```diff
@@ -73,13 +111,17 @@ class SwitchRules {
     }
 
     List<FlowRuleEntity> getServer42FlowRules() {
```

Comment:
Here and below: may be the better naming for these methods is something like 'SwitchRules.getServer42FlowRelated()`?